tests/org.jboss.tools.jst.ui.bot.test/src/org/jboss/tools/jst/ui/bot/test/nodejs/NodeJSLauncherTest.java

```diff
@@ -57,25 +55,22 @@ public static void cleanup() {
 
 	@Test
 	public void testNodeJSRunAsLauncherAvailable() {
-		ExplorerItem serverJS = new ProjectExplorer().getProject(TEST_APP_NAME).getProjectItem("server", "server.js");
-		serverJS.select();
+		ExplorerItem indexJS = new ProjectExplorer().getProject(TEST_APP_NAME).getProjectItem("index.js");
+		indexJS.select();
 		assertTrue("'Run As -> Node.js Application' not available!", runAsNodeJSAppMenu().isEnabled());
 	}
 
 	@Test
 	public void testNodeJSAppIsRunning() {
-		ExplorerItem serverJS = new ProjectExplorer().getProject(TEST_APP_NAME).getProjectItem("server", "server.js");
-		serverJS.select();
+		ExplorerItem indexJS = new ProjectExplorer().getProject(TEST_APP_NAME).getProjectItem("index.js");
+		indexJS.select();
 		runAsNodeJSAppMenu().select();
 
 		ConsoleView console = new ConsoleView();
 		console.activate();
-		new WaitUntil(new ConsoleHasText("Hello From Node.js App"));
-		new WaitUntil(new ConsoleHasText("Express server listening on port 3000"));
-
-		assertTrue("Node.js App is not running!", console.getConsoleText().contains("Hello From Node.js App"));
+		new WaitUntil(new ConsoleHasText("Listening on port 3000"));
 		assertTrue("Node.js App is not running!",
-				console.getConsoleText().contains("Express server listening on port 3000"));
+				console.getConsoleText().contains("Listening on port 3000"));
```

Comment:
I think this assert is redundant... If console does not contain the text at this point it has already failed on the wait on line 71.